DecipherNow/gm-fabric-dashboard
# Feat: Make Threads view router driven

Resolves #612 

## Proposed Changes

  - Implemented the same methods from Fabric Grid to make this component router driven
  - Moved logic from index.js to ThreadsGrid.js to follow new file pattern
  
## How to Test

  - Navigate to JVM > Threads
  - Type something in search and confirm it persists to the url bar
  - Set Group By to State and confirm it persists to the url bar
  - Set Sort By to Name and confirm it persists to the url bar
  - Hard refresh browser and make sure your settings persist
